src/components/modules/ui.ts

```diff
@@ -150,27 +152,50 @@ export default class UI extends Module {
     /**
      * Append SVG sprite
      */
-    await this.appendSVGSprite();
+    this.appendSVGSprite();
 
     /**
      * Make toolbar
      */
-    await this.Editor.Toolbar.make();
+    this.Editor.Toolbar.make();
```

Comment:
making a toolbar also can be placed at toggleReadOnly — we don't need it in read-only mode.

Reply:
I thought the same, but I'm not sure how to make and un-make the toolbars considering read-only mode can be toggled. I figured it was best to just make them, and call `closeAllToolbars` when `readOnlyEnabled === true`